src/message_publisher.cpp

```diff
@@ -312,7 +311,20 @@ void MessagePublisher::defineRosStandardPublishers(ros::NodeHandle& ref_ros_node
     ROS_WARN("SBG_DRIVER - [Publisher] SBG Mag data output are not configured, the standard Magnetic publisher can not be defined.");
   }
 
-  if (m_SbgAirData_pub_)
+  //
+  // We need either Euler or quat angles, and we must have Nav and IMU data to
+  // compute Body and angular velocity.
+  //
+  if ((m_sbgEkfEuler_pub_ || m_sbgEkfQuat_pub_) && m_sbgEkfNav_pub_ && m_sbgImuData_pub_)
```

Comment:
Why do you need IMU data?

Reply:
IMU data is needed for angular velocity which is part of the TwistStamped message returned by imu/velocity topic.